manifester/commands.py

```diff
@@ -60,8 +60,13 @@ def delete(allocations, all_, remove_manifest_file):
                 uuid=allocation.get("uuid")
             )
             if remove_manifest_file:
+                manifester_directory = (
+                    Path(os.environ["MANIFESTER_DIRECTORY"])
```

Comment:
Might want to slap on a resolve here
```suggestion
                    Path(os.environ["MANIFESTER_DIRECTORY"]).resolve()
```